specification/translation/Azure.AI.DocumentTranslation/client.tsp

```diff
@@ -41,49 +41,131 @@ interface SingleDocumentTranslationClient {
   documentTranslate is DocumentTranslation.DocumentTranslationOperations.documentTranslate;
 }
 
-// model overrides
-@@access(DocumentTranslation.DocumentTranslateResult,
+@@clientName(DocumentTranslation.DocumentTranslateBody.body,
+  "documentTranslateContent",
+  "csharp"
+);
+
+@@clientName(DocumentTranslation.DocumentTranslateBody.body,
+  "documentTranslateContent",
+  "java"
+);
+
+@@clientName(DocumentTranslation.StartTranslationBody.body,
+  "startTranslationDetails",
+  "csharp"
+);
+
+@@clientName(DocumentTranslation.StartTranslationBody.body,
+  "startTranslationDetails",
+  "java"
+);
+
+@@access(DocumentTranslationClient.getSupportedFormats,
```

Comment:
If you want to go with a purely generated python library then you shouldnt be making operations internal

Reply:
From @kristapratico: The old API had two separate endpoints for document and glossary formats. The 2024-05-01 version collapsed this under one. We hide it so we can continue to represent it as separate methods and not break users